packages/aws-library/src/aws_library/ec2/_client.py

```diff
@@ -66,10 +69,10 @@ async def ping(self) -> bool:
     @ec2_exception_handler(_logger)
     async def get_ec2_instance_capabilities(
         self,
-        instance_type_names: set[InstanceTypeType],
+        instance_type_names: set[InstanceTypeType] | None = ALL,
```

Comment:
Question: why not `instance_type_names: set[InstanceTypeType] | ALL = ALL,` ?

Reply:
I changed to a Literal. It is indeed confusing. I also dropped the default. This way I force the user to be precise on the wished outcome